apps/dashboard/src/app/(main)/(protected)/projects/[projectId]/metrics/page-client.tsx

```diff
@@ -0,0 +1,120 @@
+'use client';
+
+import { Area, AreaChart, XAxis } from 'recharts';
+import { PageLayout } from "../page-layout";
+
+import { ChartContainer, ChartTooltip, ChartTooltipContent, type ChartConfig } from "@/components/ui/chart";
+import { stackAppInternalsSymbol, useStackApp } from '@stackframe/stack';
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@stackframe/stack-ui';
+import { useEffect, useState } from 'react';
+
+
+interface LineChartDisplayConfig {
```

Comment:
we use `type` over `interface`
```suggestion
type LineChartDisplayConfig = {
```